cli/cmd/install.go

```diff
@@ -289,14 +387,12 @@ func render(w io.Writer, values *l5dcharts.Values, crds bool, valuesOverrides ma
 		}
 	}
 
-	if !crds {
-		overrides, err := renderOverrides(vals, false)
-		if err != nil {
-			return err
-		}
-		buf.WriteString(yamlSep)
-		buf.WriteString(string(overrides))
+	overrides, err := renderOverrides(vals, false)
```

Comment:
if you consolidate the render* functions, I think we'd need to move this crd logic up into the installCrds function